l2geth/core/rollup_fee_test.go

```diff
@@ -23,9 +23,14 @@ func TestCalculateRollupFee(t *testing.T) {
 			data := make([]byte, 0, tt.dataLen)
 			fee := CalculateRollupFee(data, tt.gasUsed, big.NewInt(tt.dataPrice), big.NewInt(tt.executionPrice))
 
-			dataFee := uint64((RollupBaseTxSize + len(data)) * int(tt.dataPrice))
+			zeroes, ones := zeroesAndOnes(data)
+			zeroesCost := zeroes * 4
+			onesCost := (96 + ones) * 16
```

Comment:
Should this use the constants as well? Or intentionally covering the fact that the constants may change

Reply:
Yeah this was intentional